django/conf/project_template/project_name/settings.py

```diff
@@ -17,11 +17,10 @@
 # See https://docs.djangoproject.com/en/{{ docs_version }}/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '{{ secret_key }}'
```

Comment:
I'm not replacing `SECRET_KEY`, im introducing a new option `SECRET_KEY_FILE`, and making that the default.  Those who wish to continue to use `SECRET_KEY` may do so.